src/gov/nasa/worldwind/render/SurfaceText.java

```diff
@@ -461,6 +480,9 @@ protected void applyDrawTransform(DrawContext dc, SurfaceTileDrawContext sdc)
 
         // Apply the scaling factor to draw the text at the correct geographic size
         gl.glScaled(this.scale, this.scale, 1d);
+
+        // Apply rotation angle
+        gl.glRotated(-this.rotationAngle, 0, 0, 1);
```

Comment:
If you change the name and type this becomes:

    gl.glRotated(-this.heading.degrees, 0, 0, 1);